pkg/exchange/okex/stream_test.go

```diff
@@ -0,0 +1,121 @@
+package okex
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/c9s/bbgo/pkg/exchange/okex/okexapi"
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+func getTestClientOrSkip(t *testing.T) *Stream {
+	if b, _ := strconv.ParseBool(os.Getenv("CI")); b {
+		t.Skip("skip test for CI")
+	}
+
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+		return nil
+	}
+	e := New(key, secret, passphrase)
+
+	return NewStream(e.client)
+}
+
+func TestStream(t *testing.T) {
+	t.Skip()
```

Comment:
If this unit test must be skipped, I think we should not commit it.

Reply:
ok, but one unit test file really help a lot

Reply:
We verify our code's behavior through unit tests. If a unit test is skipped, it essentially verifies nothing. I'm confused about what TestStream is supposed to verify or help. Could you explain the details?

Reply:
Originally I thought we don't verify code through unit test, since some exchange have unit test, some do not, so I assume unit test just assist writer debug code in local, I thought we just verify our code in Github.
If we really verify our unit test in Jenkins flow, we should verify its format, not just print out, or entirely not commit any test file.